src/Public/LogRhythm/Case/Test-LrUserIdFormat.ps1

```diff
@@ -0,0 +1,62 @@
+using namespace System
+using namespace System.IO
+using namespace System.Collections.Generic
+
+Function Test-LrUserIdFormat {
```

Comment:
User IDs (user number) can only be integers.  It would be fair to allow the user to specify either the name of the user OR the user number.  

Leave this function as-is for now, and I will update `Get-LrUserNumber` to become `Get-LrUser`.  The idea would be that you could pass either an integer or a string to that function, and if it returns a user, you know it's valid.